routes/web.php

```diff
@@ -23,6 +23,7 @@
 
     // pages
     Route::resource('device-groups', 'DeviceGroupController');
+    Route::resource('poller-groups', 'PollerGroupController');
```

Comment:
It's not really a resource route now, missing the store/show/edit/update functions

Reply:
change it to Route::get ?

Reply:
Yes, also, instead of using input use path it is nicer for navigation.

`Route::resource('poller-groups/{tab}', 'PollerGroupController');`

or multiple controllers:
```
Route::resource('poller-groups/log', 'PollerGroupLogController');
Route::resource('poller-groups/performance', 'PollerGroupPerformanceController');
Route::resource('poller-groups/poller', 'PollerGroupPollerController');
```

Reply:
changed to `Route::get('poller-groups/{tab}', 'PollerGroupController@index');`